charts/kubescape-cloud-operator/values.yaml

```diff
@@ -583,101 +582,51 @@ nodeAgent:
     - name: configmap-volume
       configMap:
         name: node-agent-config-map
-    - emptyDir: {}
-      name: root-falco-fs
     - hostPath:
-        path: /boot
-        type: ""
-      name: boot-fs
+        path: /
+      name: host
     - hostPath:
-        path: /lib/modules
-        type: ""
-      name: lib-modules
-    - hostPath:
-        path: /usr
-        type: ""
-      name: usr-fs
-    - hostPath:
-        path: /etc
-        type: ""
-      name: etc-fs
+        path: /run
+      name: run
     - hostPath:
-        path: /dev
-        type: ""
-      name: dev-fs
+        path: /sys/fs/cgroup
+      name: cgroup
     - hostPath:
-        path: /var/run/docker.sock
-        type: ""
-      name: docker-socket
-    - hostPath:
-        path: /run/containerd/containerd.sock
-        type: ""
-      name: containerd-socket
-    - hostPath:
-        path: /run/crio/crio.sock
-        type: ""
-      name: crio-socket
+        path: /lib/modules
+      name: modules
     - hostPath:
-        path: /proc
-        type: ""
-      name: proc-fs
+        path: /sys/fs/bpf
+      name: bpffs
     - hostPath:
         path: /sys/kernel/debug
-        type: ""
       name: debugfs
+    - emptyDir:
+      name: data
 
-  initContainer:
-    name: falco-driver-loader
-    image:
-      repository: docker.io/falcosecurity/falco-driver-loader
-      tag: 0.34.1
-      pullPolicy: IfNotPresent
-    env:
-      - name: FALCO_BPF_PROBE
-    volumeMounts:
-      - mountPath: /root/.falco
-        name: root-falco-fs
-        readOnly: false
-      - mountPath: /host/proc
-        name: proc-fs
-        readOnly: true
-      - mountPath: /host/boot
-        name: boot-fs
-        readOnly: true
-      - mountPath: /host/lib/modules
-        name: lib-modules
-      - mountPath: /host/usr
-        name: usr-fs
-        readOnly: true
-      - mountPath: /host/etc
-        name: etc-fs
-        readOnly: true
   containers:  
     nodeAgent:
       name: node-agent
-      image: 
-        repository: quay.io/kubescape/node-agent
-        tag: v0.1.75
-        pullPolicy: IfNotPresent
-      env:  
-        - name: CONFIG_ENV_VAR
-          value: "/etc/node-agent/configuration/ConfigurationFile.json"
+      image:
+        repository: quay.io/matthiasb_1/node-agent
```

Comment:
Well... we need to use the image from `quay.io/kubescape`

Reply:
yes, once released I'll update the chart

Reply:
Dont forget to add the `release` label.
Also, you probably need to update the CI.
Maybe for now, it is better if you re-tag and push